pyproject.toml

```diff
@@ -26,6 +26,7 @@ black = "^22.3.0"
 flake8 = "^4.0.1"
 isort = "^5.10.1"
 pre-commit = "^2.18.1"
+humanfriendly = "^10.0"
```

Comment:
any reason for including this new dep?

btw, adding a new dep to a project could be very impacting .. 
be very careful adding a new dep to a project, normally you should have a very good reason for that.